template.yaml

```diff
@@ -136,10 +136,11 @@ Resources:
             Resource: !Sub 'arn:aws:es:${AWS::Region}:${AWS::AccountId}:domain/*'
       ElasticsearchClusterConfig:
         DedicatedMasterEnabled: true
-        DedicatedMasterType: t2.small.elasticsearch
-        InstanceCount: 1
+        DedicatedMasterType: t2.medium.elasticsearch
```

Comment:
this works fine for now. We can test with small again after we have something working in DEV